### PR TITLE
Ds user story 13

### PR DESCRIPTION
**Pull request by danielpselph**
## All Submissions:

* [X] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [X] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [ ] What stories does the PR include
13-16
* [X] What issues does the PR
Completes user login and logout functionality
### New Feature Submissions:

* [X] Have you added an explanation of what your changes do?
* [X] Have you lint your code locally prior to submission?

### Testing:

* [X] Have you written new tests for your core changes, as applicable?
* [X] Have you successfully ran tests with your changes locally?
* [X] All new and existing tests passed.

**Test coverage** = 100%
--------------------------------------------------------------------------------

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,7 +2,15 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   get "/", to: "welcome#index"
-  get "/login", to: "sessions#index"
+
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
+
+  # resources :sessions, only: [:new, :create, :destroy]
```
Comment (Mariana-21) [inline config/routes.rb:10]: Might want to clean this up

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+            else current_regular?
+                redirect_to profile_path
+            end
+        flash[:notice] = "You are already logged in!"
+        end
     end
-end
\ No newline at end of file
+
+    def create
```
Comment (tylertomlinson) [inline app/controllers/sessions_controller.rb:16]: I like how this logic makes it so we don't need to create a welcome method. Also I like the idea of have a helper method for the routing. Lets look into that soon

File: spec/features/users/login_spec.rb [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+            click_button "Sign In"
+
+            visit "/login"
+
+            expect(page).to have_content("You are already logged in!") 
+            expect(current_path).to eq(admin_path) 
+        end
+    end
+end
```
Comment (tylertomlinson) [inline spec/features/users/login_spec.rb:156]: Nice edge case testing. Very robust.

**Review by tylertomlinson — APPROVED** [review]

**Review by Mariana-21 — APPROVED** [review]
Really robust testing! other than the commented out resources this looks great!!

**Pull request merged**